language/e2e-tests/src/tests/genesis.rs

```diff
@@ -36,3 +37,23 @@ fn invalid_genesis_write_set() {
         VMStatus::new(StatusCode::INVALID_WRITE_SET)
     );
 }
+
+#[test]
+fn execute_genesis_write_set() {
+    let executor = FakeExecutor::no_genesis();
+    // Genesis write sets are not allowed to contain deletions.
+    let write_set = GENESIS_WRITE_SET.clone();
+    let address = account_config::association_address();
+    let (private_key, public_key) = compat::generate_keypair(None);
+    let txn = transaction_test_helpers::get_write_set_txn(
+        address,
+        0,
+        &private_key,
+        public_key,
+        Some(write_set),
+    )
+    .into_inner();
+
+    // We don't have a verification story for WriteSet yet. Just run the Executor for now.
```

Comment:
Should we kill this comment? I don't think we will have a verification story in the future given that write sets bypasses the bytecode verifier and VM.